doc/packer.txt

```diff
@@ -531,6 +540,15 @@ It can be invoked with no arguments or with a list of plugin names to update.
 These plugin names must already be managed by `packer` via a call to
 |packer.use()|.
 
+snapshot()		                                *packer.snapshot()*
+`snapshot` takes the rev of all the installed plugins and serializes them into a lua table which will be saved at `config.snapshot_path` or an absolute path provided by the users. The snapshot can be loaded via `dofile`
```

Comment:
Nitpick: please capitalize "Lua".

Reply:
We might also want to clarify that `config.snapshot_path` is a directory holding multiple snapshot files, and the actual path to which the snapshot is saved has some additional path component.